crypto/fipsmodule/rand/rand.c

```diff
@@ -262,9 +264,11 @@ static void rand_thread_state_free(void *state_in) {
   CRYPTO_STATIC_MUTEX_unlock_write(thread_states_list_lock_bss_get());
 
   rand_state_fips_clear(state);
+#else
+  OPENSSL_cleanse(state, sizeof(struct rand_thread_state));
```

Comment:
This could really be called for both fips and non-fips code path. But it's fine as-is now, since the drbg state is zeroised. in both paths.